src/shared/services/export-favorites/export-favorites.utils.ts

```diff
@@ -46,24 +43,25 @@ import { SLASH, CYPHER_FILE_EXTENSION } from './export-favorites.constants'
  * @return    {Object[]}                        new user favorites objects ("my scripts")
  */
 export function mapOldFavoritesAndFolders(
-  favorites: any,
-  folders: any,
-  isAllowed = isNonStatic
+  favorites: Favorite[],
+  folders: Folder[],
+  isAllowed: (fav: Favorite) => boolean = isNonStatic
 ) {
   const oldFoldersMap = new Map(
+    // @ts-ignore
```

Comment:
what is being ignored?

Reply:
Since we'll be changing how export works I didn't spend any time fixing it for now!